draft/security.md

```diff
@@ -0,0 +1,109 @@
+# Security Working Group
+
+🚧 Draft charter: this working group isn't currently approved. The proposal is for establishing a formal security team to coordinate Django's security efforts.
+
+This charter proposes the formation of an official Security Working Group to formalize and expand the security coordination efforts that have been part of Django's development process. The Security Working Group will be responsible for ensuring that Django and associated projects maintained by the Django Software Foundation maintain high security standards and respond effectively to security incidents.
+
+## Responsibilities
+
+This working group will focus on security aspects of the Django project. The team will be responsible for addressing security concerns, reviewing vulnerability reports, and providing guidance on security best practices throughout the Django ecosystem.
+
+The duties of the working group are:
+- Receive, triage, and respond to security vulnerability reports for Django
+- Coordinate the release of security patches with the Django release team
+- Review security-related pull requests across Django projects
+- Provide security guidance on Django features and APIs
+- Monitor the security landscape to proactively identify potential vulnerabilities
+- Help maintain Django's security documentation
+- Develop and maintain security-related packages in the Django ecosystem
+- Provide security advice to Django package maintainers when requested
+- Assist with security aspects of the djangoproject.com website
+- Mentor new contributors interested in Django security
+- Run or support security-focused sessions in DjangoCon sprints
+- Chair, Co-Chair and Board Liaison can sign off on security decisions
+
+## Initial membership
+
+- Chair: TBD
+- Co-Chair: TBD
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD
+- Other members:
+  - TBD
+  - ...
```

Comment:
I assume this will be the [Security team members](https://www.djangoproject.com/foundation/teams/#security-team) as-is?

Reply:
I think it should be implied that the Security Team sit on (or at least have input into) the WG.

Reply:
This is perhaps a good opportunity to check with the existing members whether they'd like to continue their position.

I think at least for the migration to a WG, we shouldn't switch members too much during the transition.

Reply:
We did a round of that at the end of last year, so I don't think we need to do another check-in quite so soon.